indico/modules/events/papers/util.py

```diff
@@ -83,3 +85,14 @@ def get_user_submittable_contributions(event, user):
     return (_query_contributions_with_user_paper_submission_rights(event, user)
             .filter(db.or_(*criteria))
             .all())
+
+
+def is_type_reviewing_possible(cfp, review_type):
+    if review_type == PaperReviewType.content:
+        return (cfp.content_reviewing_enabled
+                and (not cfp.content_reviewer_deadline_enforced or cfp.content_reviewer_deadline > now_utc()))
+    elif review_type == PaperReviewType.layout:
+        return (cfp.layout_reviewing_enabled
+                and (not cfp.layout_reviewer_deadline_enforced or cfp.layout_reviewer_deadline > now_utc()))
+    else:
+        return False
```

Comment:
is that's never supposed to happen, better raise an exception.. `ValueError('invalid review type')` maybe